### PR TITLE
fix(FTL-8754): implemented Cognito Lambda error handling

### PR DESCRIPTION
**Pull request by oleksiipiliugin**
- `CompleteLoginPasswordlessResult` was [extended](https://github.com/affinidi/affinidi-core-sdk/pull/284/files#diff-fb96193d1cb2d478423b97182b3578d3965f49d08a8104e5d4465af339fbad7fR46) with `VerifyAuthLambdaCustomError` in order to handle Cognito Lambda error handling

### CODE REVIEW

File: sdk/core/test/integration/otp/core.test.ts [hunk 1]
```diff
@@ -733,5 +733,20 @@ parallel('CommonNetworkMember [OTP]', () => {
 //       const { wallet } = await AffinidiWallet.signInWithProfile(options, trueCallerTestProfile)
 //       checkIsWallet(wallet)
 //     })
+//
```
Comment (anton-iskryzhytskyi) [inline sdk/core/test/integration/otp/core.test.ts:736]: why it is commented?

**Review by anton-iskryzhytskyi — APPROVED** [review]
LGTM, minor question

File: sdk/core/test/integration/otp/core.test.ts (hunk 1 above)
Comment (oleksiipiliugin) [reply sdk/core/test/integration/otp/core.test.ts:736]: it is temporarily blocked by infrastructure issues. will be uncommented after it will be resolved

Comment (linn-affinidi) [reply sdk/core/test/integration/otp/core.test.ts:736]: if it is temp, i think we should use it.skip()

**Pull request merged**